index.js

```diff
@@ -56,6 +79,38 @@ function assembleStyles () {
 		});
 	});
 
+	styles.color = {
+		ansi16: {},
+		ansi256: {},
+		ansi16m: {}
+	};
+
+	styles.bgColor = {
+		ansi16: {},
+		ansi256: {},
+		ansi16m: {}
+	};
+
+	for (var key in colorConvert) {
+		var fn = colorConvert[key];
+		var conversion = key.replace(/^([^2]+).+$/, '$1');
+		if (/2ansi16$/.test(key)) {
+			styles.color.ansi16[conversion] = wrapAnsi16(fn, 0);
```

Comment:
Could either be

- `ansi16` -> `ansiBasic`
- `ansi16` -> `ansi`
- `ansi16m` -> `ansiTruecolor`